app/MMB/src/scenes/FlightServices.js

```diff
@@ -91,6 +92,16 @@ export default class FlightServices extends React.Component<Props> {
   );
 }
 
+type Props = {|
+  +navigation: NavigationType,
+|};
+
+export default function FlightServicesWithContext(props: Props) {
+  <BookingDetailContext.Consumer>
+    {({ bookingId }) => <FlightServices {...props} bookingId={bookingId} />}
```

Comment:
Similar here - just looks like a special kind of prop-drilling. Navigation prop-drilling... :)